formwidgets/menuitems/assets/js/menu-items-editor.js

```diff
@@ -81,11 +81,19 @@
             self.$popupForm = self.$popupContainer.find('form')
             self.itemSaved = false
 
-            $('input[name=title]', self.$popupContainer).focus().select()
-            $('select[name=type]', self.$popupContainer).change(function(){
+            var titleField = $('input[name=title]', self.$popupContainer).focus().select()
+            var typeField = $('select[name=type]', self.$popupContainer).change(function(){
```

Comment:
minor quibble here, shouldn't variables containing jQuery objects be named prefixed with `$`? i.e. `$titleField` and `$typeField`?